docs/application/native/guides/location-sensors/device-sensors.md

```diff
@@ -407,7 +407,25 @@ The following table lists the measurement data that the HRM LED green sensor pro
 | Measurement                      | Type                 | Range                                   | Unit         |
 |----------------------------------|----------------------|-----------------------------------------|--------------|
 | Timestamp                        | `unsigned long long` | -                                       | Microseconds |
-| values[0]: HRM green light value | `int`                | Min. value = 0<br> Max. value = 1081216 | -            |
+| values[0]: HRM green light value | `int`                | Min. value = 0<br> Max. value = 4194304 | -            |
+
+<a name="hrm_green_batch"></a>
+## Heart Rate Monitor LED Green batch Sensor
+
+The Heart Rate Monitor (HRM) LED green batch sensor measures the amount of green light that is reflected back from a person's blood vessel. The sensor must be power-efficient and work in a batch manner.
+
+The following table lists the measurement data that the HRM LED green batch sensor provides.
```

Comment:
Global:

@bbrto21  I think you have missed my comment 
The lines such as 
 The following table lists the measurement data that the HRM LED green batch sensor provides.

```suggestion
The following table lists the measurement data that the HRM LED green batch sensor provides:
```
You need to replace the period with the colon because a table follows up. 
Grammatically, a colon does the job of directing you to the information following it.

Reply:
@nawab-samsung 
Should I replace the period with the colon? 
As you mentioned above, there are the same things throughout this document. Should I replace them all?

Reply:
@bbrto21 yes kindly replace . with : 
Ideally yes, that's the best practice to follow, as the sentence is followed by table so : must be used.

Reply:
I changed all the instances.

Reply:
@bbrto21  thanks, will check and let you know if any further changes are needed.